malaria-app-android/src/main/java/com/peacecorps/malaria/activities/SideEffectsNPCVFragmentActivity.java

```diff
@@ -155,6 +169,10 @@ private void hidepDialog(){
         //for dismissing the loading animation in Activity
     }
 
-
+    @Override
+    protected void onSaveInstanceState(Bundle outState) {
```

Comment:
I think it would be better to avoid using savedInstanceState at all in any of the InfoHub Activity, since restarting the activity on orientation change is not required here. As stated on the developer.android.com  "If your application doesn't need to update resources during a specific configuration change and you have a performance limitation that requires you to avoid the activity restart, then you can declare that your activity handles the configuration change itself, which prevents the system from restarting your activity." 
Here I guess, we are using the the same resources for both the layouts so updating resources is not required.